dev/buildtool/cloudbuild/containers-build-java8.yml

```diff
@@ -1,15 +1,26 @@
 steps:
+  - id: restoreCache
+    name: $_DOCKER_REGISTRY/restore_cache:latest
+    args:
+      - "--bucket=gs://spinnaker-build-cache"
```

Comment:
Probably want to parameterize this and just set a default

Reply:
Good idea, thanks.